robottelo.properties.sample

```diff
@@ -2,6 +2,10 @@
 [server]
 # Server hostname
 hostname=
+# Additional satellite hostnames for xdist workers
+# gw0=
```

Comment:
Wouldn't it be cleaner to make `hostname` list of available hostnames?

Then helper function can extract number from worker name, get modulo of that on list length and use that as index:

```python
>>> worker_id = "gw5"
>>> hosts = [f'dhcp{num}.example.com' for num in range(4)]
>>> hosts
['dhcp0.example.com', 'dhcp1.example.com', 'dhcp2.example.com', 'dhcp3.example.com']
>>> worker_num = int(worker_id[2:])  # or something more clever, if workers aren't always named `gw<int>`
>>> hosts[ worker_num % len(hosts) ]
'dhcp2.example.com'
```

This helper should also account for `settings.server.hostname` being either string or a list.

Reply:
that was similar to my initial thought, but this is more clear for ini-based config files.

Reply:
Wouldn't say one is clearer than another:

```
[server]
hostname=dhcp0.example.com,dhcp1.example.com,dhcp2.example.com
```

```
[server]
hostname=dhcp0.example.com
gw0=dhcp1.example.com
gw1=dhcp2.example.com
```

But I would say having single setting that is list of values is preferable to leaking internal pytest-xdist identifiers to settings module.

This is ultimately up to you, but we could have solution that is clean and nicely integrated with dynaconf from day one.